actions/run-tunnel-server/action.yaml

```diff
@@ -0,0 +1,92 @@
+name: Run tunnel server
+description: Run tunnel server solution and return order numbers
+
+inputs:
+  tee_offers:
+    description: Tee offers separated by comma (eg. 1,3)
+    required: true
+  target:
+    description: Specify "develop" "stage" or "testnet"
+    required: true
+  previous_orders:
+    description: Previous order numbers to cancel, separated by comma (eg. 100,102)
+  min_rent_minutes:
+    description: How long order should run
+  auth_token:
+    description: The authentication token used by the tunnel server to identify tunnel clients
+    required: true
+
+outputs:
+  order_ids:
+    description: Created tunnel server order ids
+    value: ${{ steps.create-workflows.outputs.order_ids }}
+
+runs:
+  using: composite
+  steps:
+    - name: Check spctl
+      uses: Super-Protocol/sp-build-tools/actions/check-spctl@start-tunnels-actions
+
+    - name: Check variables
+      shell: bash
+      run: |
+        if [ "${{ inputs.target }}" != "develop" ] && [ "${{ inputs.target }}" != "stage" ] && [ "${{ inputs.target }}" != "testnet" ]; then
+          echo "Invalid target type: ${{ inputs.target }}"
+          exit 1
+        fi
+
+    - name: Set offers by target
+      shell: bash
+      run: |
+        if [ "${{ inputs.target }}" == "develop" ]; then
+          BASE_IMAGE_OFFER=6
+          TUNNEL_SERVER_OFFER=11
+          STORAGE_OFFER=25
+        elif [ "${{ inputs.target }}" == "stage" ]; then
+          BASE_IMAGE_OFFER=3
+          TUNNEL_SERVER_OFFER=8
+          STORAGE_OFFER=22
+        elif [ "${{ inputs.target }}" == "testnet" ]; then
+          BASE_IMAGE_OFFER=6
+          TUNNEL_SERVER_OFFER=11
+          STORAGE_OFFER=25
+        fi
+
+        echo "BASE_IMAGE_OFFER=$BASE_IMAGE_OFFER" >> $GITHUB_ENV
+        echo "TUNNEL_SERVER_OFFER=$TUNNEL_SERVER_OFFER" >> $GITHUB_ENV
+        echo "STORAGE_OFFER=$STORAGE_OFFER" >> $GITHUB_ENV
+
+    - name: Prepare working directory
+      shell: bash
+      run: |
+        WORKDIR=$RUNNER_TEMP/${{ github.run_id }}/tunnel-server-data
+        mkdir -p $WORKDIR/archive
```

Comment:
the same commanet as [here](https://github.com/Super-Protocol/sp-build-tools/pull/2/files#r1784055229)